ledger-core/v2/virtual/pulsemanager/pulsemanager.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package pulsemanager
+
+import (
+	"context"
+	"sync"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/v2/log"
+	"github.com/insolar/assured-ledger/ledger-core/v2/network"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
+
+	"github.com/pkg/errors"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/flow/dispatcher"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/jet"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/node"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/pulse"
+)
+
+// PulseManager implements insolar.PulseManager.
+type PulseManager struct {
```

Comment:
pulsemanager.Service?

Reply:
PulseManager is not a service for VM and it'll be gone

Reply:
it is Service in package puslemanager. Common case for now if it is service that represents subsystem realization, it should be called Service. So it would be pulsemanager.Service, not puslemanager.Pulsemanager (which is tautology)